### ISSUE TITLE
Sub-generator targets not imported.

### ISSUE THREAD

**Issue by chiefy** [balderdashy/sails-generate#4]
Lets say I have a generator I've installed globally called `sails-generate-new-awesome`, this generator has it's own dependencies setup in its `package.json`

```
  "dependencies": {
    "async": "~0.2.9",
    "lodash": ">=2.4.x",
    "merge-defaults": ">=0.1.0",
    "pluralize": "~0.0.11",
    "sails-generate-backend-awesome": "~0.0.1",
    "sails-generate-frontend-awesome": "~0.0.1"
  }
```

Currently if I try to use these dependencies as targets without installing those dependencies globally, it will die: (index.js) 
```javascript
   targets: {
        '.': ['backend-awesome', 'frontend-awesome'],
    ....
```

I believe I have fixed this issue, and will be submitting a PR, but just wanted to document it here.

**Comment by CWyrtzen** [comment]
Hi @chiefy is this still an issue?

**Comment by sailsbot** [comment]
Thanks for posting, @chiefy.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
